<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagsCompatibilityOptionsPage</class>
 <widget class="QWidget" name="TagsCompatibilityOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>705</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="aac_tags">
     <property name="title">
      <string>AAC files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="aac_save_ape">
        <property name="text">
         <string>Save APEv2 tags</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="aac_no_tags">
        <property name="text">
         <string>Do not save tags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="remove_ape_from_aac">
        <property name="text">
         <string>Remove APEv2 tags from AAC files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>274</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
